### PR TITLE
build: add midudev template

### PR DESCRIPTION
**Pull request by midudev**
- [x] Add https://midu.dev template.
- [x] Sort import lines because of my OCD. 😆

### CODE REVIEW

File: src/components/presets/midudev.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import { Image, Box, Link, Flex, Text } from './scope'
+
+const code = (
+  <Inline>
+    <>
+      <Link
+        href='https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap'
+        rel='stylesheet'
+      />
+      <Box as='header' sx={{}}>
```
Comment (Kikobeats) [inline src/components/presets/midudev.js:13]:
```suggestion
      <Box as='footer'>
```

Maybe that was your original intention? 😛

File: src/components/presets/midudev.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+              alignItems: 'center',
+              margin: '0 auto'
+            }}
+          >
+            <Image
+              sx={{
+                marginBottom: '32px',
+                width: '164px'
+              }}
+              src={`https://midu.dev/images/tags/${query.tag}.png`}
```
Comment (Kikobeats) [inline src/components/presets/midudev.js:78]: smart 👏

File: src/components/presets/midudev.js (hunk 1 above)
Comment (Kikobeats) [reply src/components/presets/midudev.js:13]: it's worth it to move after the main component to be real footer!

Comment (midudev) [reply src/components/presets/midudev.js]: 👍Moved to the footer! I started with the logo on the top of the image and I forgot to change it. :)

**Pull request merged**